packages/plugin-ext/src/main/browser/style/tree.css

```diff
@@ -30,12 +30,7 @@
     margin-left: var(--theia-ui-padding);
 }
 
-.theia-TreeNodeSegment {
```

Comment:
that's for sure was bogus to change all tree styles in the plugin system 👍 

but we should check that it does not cause any side effects first on the plugin trees and then for any other tree which could rely on it

Reply:
It was introduced by this commit https://github.com/eclipse-theia/theia/commit/65d5925d450d529be45902b23f421d393211d38c, not so long time ago. @svenefftinge Do you remember which UI glitch it fixed?

So probably only the plugin tree is affected, maybe instead of removing it prepend with `.theia-tree-view`.

Reply:
> So probably only the plugin tree is affected, maybe instead of removing it prepend with `.theia-tree-view`.

Thankyou for your response, I have prepended `.theia-tree-view` to the class and it works fine. 
![image](https://user-images.githubusercontent.com/43870550/72904253-96580f00-3cfc-11ea-8f8c-1e89374c0680.png)

Reply:
I think just adding `.theia-tree-view` to the existing css rules instead of removing it and re-adding it to the bottom of the file would be better.

We will also need to fix `tree-views` as they are not displayed correctly (they also do not have a consistent wrapping with ellipsis). For example, the `npm scripts` tree view:

<img width="379" alt="Screen Shot 2020-01-22 at 12 36 14 PM" src="https://user-images.githubusercontent.com/40359487/72918661-04f49700-3d14-11ea-8a36-dc7b6e007375.png">